gpu-miner.c

```diff
@@ -100,9 +109,11 @@ double grindNonces(size_t items_per_iter, int cycles_per_iter) {
 		if (ret != CL_SUCCESS) { printf("failed to write to targmobj buffer: %d\n", ret); exit(1); }
 
 		// Execute OpenCL kernel as data parallel
+		// Note that minimum intensity is 8, making the local
+		// worksize always divisible by the global worksize
```

Comment:
Do you mean the global worksize is divisible by the local worksize?
But in the lshift above you use `Intensity - 1` -- doesn't that make `GlobalSize = 128` when the Intensity is 8?